kubeportal/tests/test_api.py

```diff
@@ -51,6 +51,13 @@ def post(self, relative_url, data=None, headers={}):
         return self.client.post('http://testserver' + relative_url,
                                 json=data, headers=headers)
 
+    def options(self, relative_url, headers={}):
+        if self.jwt:
+            headers["Authorization"] = None
+        if self.csrf:
```

Comment:
Is this the same as not setting them at all? If yes, why you do it? Please add an according comment in the source code.

Reply:
well, actually yes. I just remembered funny networktools observations which led me to think that the key is necessary. In any case, it doesn't matter how the options-request looks like.